src/sentry/statistical_detectors/detector.py

```diff
@@ -234,9 +234,9 @@ def detect_regressions(
 
         for chunk in chunked(cls.all_timeseries(objects, start, function), timeseries_per_batch):
             data = {}
-            for project_id, transaction_name, result in chunk:
+            for project_id, object_name, result in chunk:
                 serialized = serializer.serialize(result, get_function_alias(function))
-                data[f"{project_id},{transaction_name}"] = {
+                data[f"{project_id},{object_name}"] = {
```

Comment:
Not sure if you intended to have these changes in this PR, but they look good to me

Reply:
Going to keep this around as the `transaction_name` naming was copied from when I refactored this